syntax="proto3";

message vect2 {
	float x=1;
	float y=2;
}

message TestMsg {
	int32 field_id = 1;
	int32 id = 2;
	vect2 direction = 3;
	int32 health = 4;
	bool can_shoot = 5;
}